pkgs/by-name/an/anyrun/package.nix

```diff
@@ -69,5 +69,6 @@ rustPlatform.buildRustPackage rec {
       NotAShelf
     ];
     mainProgram = "anyrun";
+    badPlatforms = lib.platforms.darwin;
```

Comment:
We should set platforms to linux instead.

Reply:
I was on the fence about it.. We do have a couple of linux only dependencies but the derivation was set up with conditionals for platforms already.

Reply:
Done, with note.